frontend/src/pages/bceidform/ReviewWizardStep.vue

```diff
@@ -96,7 +96,7 @@ onMounted(() => {
       <hr class="divider" v-if="index > 0" />
       <h6 class="review-icon-title">
         <LocationStar20 v-if="index === 0" />
-        <Location20 v-else />{{ toTitleCase(address.locationName) }}
+        <Location20 v-else />{{ toSentenceCase(address.locationName) }}
```

Comment:
I think we should just present the name as is here, without any conversion, since it's already converted in the exportAddress.
Also because, like you said yesterday, maybe there is an acronym in the name or any other needed upper case letter.

Reply:
We had a conversation with Andrea this morning and it was agreed to leave it like I have it